recipes-openxt/qemu-dm/qemu-dm-2.6.2/ovmf-q35.patch

```diff
@@ -0,0 +1,284 @@
+Index: qemu-2.6.2/hw/i386/pc_piix.c
+===================================================================
+--- qemu-2.6.2.orig/hw/i386/pc_piix.c
++++ qemu-2.6.2/hw/i386/pc_piix.c
+@@ -56,6 +56,8 @@
+ #include "migration/migration.h"
+ #include "kvm_i386.h"
+ 
++extern int xen_q35;
```

Comment:
I do not like blind accessible global state, they are too easily abused and leaves yourself open to potential exploit. Make it local and export wrappers to set and get that guard for correct values, etc.

Reply:
Seems like it's how QEMU defines a bunch of other Xen variables too in `include/hw/xen/xen.h`, so I would just place this next to the other ones:

```
extern uint32_t xen_domid;
extern enum xen_mode xen_mode;
extern bool xen_allowed;